apps/website/src/content/docs/flex.mdx

```diff
@@ -1,20 +1,70 @@
 ---
 title: Flex
-description: A utility component to work with CSS flexbox and iTwinUI design tokens.
+description: A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
```

Comment:
The frontmatter description needs to be kept short, ideally one paragraph.

It gets shown in the cards on this page and we should try to prevent too much truncation. https://itwinui.bentley.com/docs/components

Reply:
I have shortened the description. The last two sentences are mentioned in the sections below already so I have omitted those.